products.json

```diff
@@ -13,6 +13,20 @@
         "artifactory-token": "ARTIFACTORY_TOKEN",
         "tarball-regex": "spark-*.tgz"
     },
+    {
+        "name": "kafka",
+        "lp-building-project": "soss",
+        "lp-releasing-project": "kafka-releases",
+        "lp-building-repo": "~data-platform/soss/+source/charmed-kafka/+git/charmed-kafka/",
+        "lp-building-branch-prefix": "lp-3",
+        "lp-consumer-key": "LP_CONSUMER_KEY",
+        "lp-access-token": "LP_ACCESS_TOKEN",
+        "lp-access-secret": "LP_SECRET_TOKEN",
+        "artifactory-url": "https://canonical.jfrog.io/artifactory/dataplatform-maven-local/",
```

Comment:
I believe we should change also the others into this one, as per the spec that all the Canonical built artifacts should go into one single repository